src/test/regress/expected/multi_materialized_view.out

```diff
@@ -0,0 +1,125 @@
+---
+--- MULTI_MATERIALIZED_VIEW
```

Comment:
The multi_ is there only for historical reasons (Citus used to be in the Postgres source tree and multi_ was used to distinguish Citus files). Newer test files omit it.